ckcp/defaults.yaml

```diff
@@ -0,0 +1,26 @@
+# This YAML file allows openshift_dev_setup.sh to fetch values of certain variables.
```

Comment:
I'm not sure the name `defaults.yaml` fits the purpose of this config file very well. My understanding is that the defaults value are actually hardcoded in the script (e.g. line 93/94)
Since this file holds the config for the ckcp deployment, I would propose to rename it to `ckcp.yaml` (or `config.yaml`).

Reply:
Done.